### DISCUSSION TITLE
Show posts by tag

### DISCUSSION THREAD

**Discussion by Pepo-BG** [dailydotdev/daily#929] [Feature Requests]
Hi there!

The Chrome extension and mobile app are great!
I have just 1 feature request - availability to search by any specific tag from home page/menu on mobile app.
For example, now when I want to browse only posts, related to .net, I have to open the link `https://app.daily.dev/tags/.net`. It will be great if there's a menu with all followed tags, then select any of them, and show posts for it.

Best Regards,
Pepo

**Comment by idoshamun** [comment]
Hi @Pepo-BG,

Thanks for the suggestion! Can you elaborate on why you would like to see posts with a specific tag?

**Reply by Pepo-BG** [reply]
Hi @idoshamun ,

For example - my feed has posts with many different tags (which is awesome) based on my interests, like Web Development, Mobile Development, etc. And there are many posts (which is great, too) for these tags. But today I want to read posts for `android` only. And I need to filter them easily.

**Reply by idoshamun** [reply]
Thanks a lot for the info!

**Reply by rvrlx** [reply]
Great suggestion @Pepo-BG .
